Feverup/pysugarcrm
# Avoid misleading error when Sugar connexion is not performed successfully

[SD-1272](https://feverup.atlassian.net/browse/SD-1272)
Related issue: #16 

If trying to connect to SugarCRM ends with a exception. The finally statement, will try to access a non-existing variable (conn). This PR changes this.

### How to test:
From fever2 BO, try to create a plan with a invalid Partner/Contract ID. you should get this error:
![image](https://user-images.githubusercontent.com/33002592/132205299-88456751-2f00-43c7-99e5-2fcfea144ba0.png)
